.env.example

```diff
@@ -15,4 +15,4 @@ AWS_PROFILE=
 IN_CLUSTER=
 ENTERPRISE_API_URL=
 K1_LOCAL_DEBUG=true
-K1_LOCAL_KUBECONFIG_PATH=
+K1_LOCAL_KUBECONFIG_PATH=
```

Comment:
The reason btw this line is giving you this message is because it's a common practice to end all text files with a new line (at least in the UNIX world) so GitHub reports those files that didn't get a new line with this circle and the dash in the middle.

https://github.com/microsoft/vscode/issues/12076 this issue shows how to configure VSCode to add them automatically.